lib/reek/cli/warning_collector.rb

```diff
@@ -8,16 +8,20 @@ module CLI
     # @api private
     class WarningCollector
       def initialize
-        @warnings = Set.new
+        @warnings_set = Set.new
       end
 
       def found_smell(warning)
-        @warnings.add(warning)
+        warnings_set.add(warning)
       end
 
       def warnings
-        @warnings.to_a.sort
+        warnings_set.to_a.sort
```

Comment:
Yeah, I was unhappy about the type-based name, but I didn’t find a better solution that would keep being a minimal diff. :)